test/reviewer.test.js

```diff
@@ -139,10 +141,14 @@ describe('reviewer', function() {
           engineers: [ 'engineers', 'dr-mario' ],
           designers: [ 'designers' ],
           yoshi: [ 'mario', 'luige' ],
+          'team:koopa-troop': [ 'mario' ],
         },
       },
     };
 
+    const stub = sinon.stub(github, 'get_team_members');
+    stub.withArgs('koopa-troop').returns([ 'bowser', 'king-boo', 'goomboss' ]);
```

Comment:
Actually, this stubs expects that `get_team_members` to return a simple array, not a response described in https://docs.github.com/en/rest/teams/members?apiVersion=2022-11-28#list-team-members